samples/net/ocpp/README.rst

```diff
@@ -26,6 +26,14 @@ Requirements
 - SteVe Demo Server (<https://github.com/steve-community/steve/blob/master/README.md>)
 - LAN for testing purposes (Ethernet)
 
+Generating keys and certificate for TLS (Optional):
+***************************************************
+
+- Generate a ECDSA Private key using OpenSSL:
+  openssl ecparam -genkey -noout -out ec-priv.key
```

Comment:
You could wrap the openssl commands in `.. code-block:: console` so that it renders better in docs:
```
.. code-block:: console

    openssl ecparam -genkey -noout -out ec-priv.key
```